### PR TITLE
GEODE-9086: update docs to show support for DECRBY

### PR DESCRIPTION
**Pull request by nonbinaryprogrammer**
this corresponds with commit 39c0abf96ae93a2a49fd0ab28ad2e96c1fa9e7a8

Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [ ] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [ ] Is your initial contribution a single, squashed commit?

- [ ] Does `gradlew build` run cleanly?

- [ ] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, check Concourse for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

File: geode-redis/README.md [hunk 1]
```diff
@@ -159,53 +159,53 @@ start server \
 |-----------------------|-------------------------------------------------------|-----------------------------------|
 | APPEND             	| BITCOUNT                                           	| ACL CAT                       	|
 | AUTH               	| BITOP                                              	| ACL DELUSER                   	|
-| DEL                	| BITPOS                                             	| ACL GENPASS                   	|
-| EXISTS             	| DBSIZE                                             	| ACL GETUSER                   	|
-| EXPIRE             	| DECR                                               	| ACL HELP                      	|
```
Comment (sabbey37) [inline geode-redis/README.md:164]: Just noticed DECR is in the unsupported column here. I guess that could be changed in a different PR though.

**Review by sabbey37 — APPROVED** [review]

File: geode-redis/README.md (hunk 1 above)
Comment (nonbinaryprogrammer) [reply geode-redis/README.md:164]: we added a ticket to the backlog for double checking that the docs are correct with the state of the support/1.14 branch
